src/le.py

```diff
@@ -1167,10 +1167,10 @@ def config_formatters():
 
 def extract_token(log_):
     """Extract the log token value if it exists"""
-    try:
-        if 'log' in log_ and log_['log']['source_type'] is 'token':
-            return log_['log']['tokens'][0]
```

Comment:
The intention of the original suggestion (from a previous pull request), that resulted in this change, was to help eliminate the try/except block, and the if clause, by using the following statement:

`token = log_.get('log').get('tokens')[0]`
 
This assumes, that if the value associated with 'source type' is 'token', then the 'tokens' key will be present in the dictionary as well. If this is not a safe assumption to make, then the following could be used:

`token = log_.get('log').get('tokens')[0] if (log_['log']['source_type'] is 'token') else None`

But that's a bit dense. An easier to follow variant might be:

```
if log_['log']['source_type'] is 'token':
    token = log_.get('log').get('tokens')[0]
```

The spirit of the original suggestion, was, to leverage the built-in functionality, of the dict.get() method, to perform most of the work of the try/except block, and the if clause.